hips/tiles/tests/test_surveys.py

```diff
@@ -11,33 +11,44 @@ class TestHipsSurveyProperties:
     @classmethod
     def setup_class(cls):
         filename = get_pkg_data_filename('data/properties.txt')
-        cls.hips_survey = HipsSurveyProperties.read(filename)
+        cls.survey = HipsSurveyProperties.read(filename)
+
+    @remote_data
+    def test_from_name(self):
+        survey = HipsSurveyProperties.from_name('CDS/P/2MASS/color')
+        assert survey.title == '2MASS color J (1.23 microns), H (1.66 microns), K (2.16 microns)'
+
+    @remote_data
+    def test_make(self):
+        survey = HipsSurveyProperties.make('CDS/P/EGRET/Dif/300-500')
+        assert survey.title == 'EGRET Dif 300-500'
+        self.survey = HipsSurveyProperties.make(self.survey)
```

Comment:
This line doesn't make sense. You're setting `self.survey` on the last line of the test.
Instead you should be asserting something.
I think this might work:
```
assert self.survey is HipsSurveyProperties.make(self.survey)
```
because in that case we just return the same object and "is" checks for object equality, no?